pkg/utils/interfaces_test.go

```diff
@@ -0,0 +1,36 @@
+package utils
```

Comment:
For the test, it might be nice to make it mockable and not depend on actual values from the system, something like:
https://stackoverflow.com/questions/68940230/how-to-mock-net-interface